app/models/validations/date_validations.rb

```diff
@@ -91,4 +101,15 @@ def date_valid?(question, record)
   def is_rsnvac_first_let?(record)
     [15, 16, 17].include?(record["rsnvac"])
   end
+
+  def inactive_status(date, deactivation_periods, available_from)
+    return if date.blank?
+
+    closest_reactivation = deactivation_periods.reverse.find { |period| period.reactivation_date.present? && date.between?(period.deactivation_date, period.reactivation_date - 1.day) } if deactivation_periods.present?
```

Comment:
Do we not need to sort the reactivation periods before reversing them? I couldn't find sorting being applied anywhere.

Reply:
Hm, I think this might have been linter. I suspect I've had something like:
`deactivation_periods.select { |period| period.reactivation_date.present? && date.between?(period.deactivation_date, period.reactivation_date - 1.day) }.last `

My assumption was that the records are returned ordered by `created_at` date, but even if it's so I haven't actually found any proof for that, so I updated the statement to explicitly order it. Thanks!